### PR TITLE
Modified the docs of resource_group_policy_assignment

### LINKED ISSUE #12542: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group_policy_assignment is incorrect
The code sample on this page is incorrect:
```
resource "azurerm_resource_group_policy_assignment" "example" {
  name                 = "example"
  resource_group_id    = azurerm_resource_group.example.id
  policy_definition_id = azurerm_policy_assignment.example.id
}
```

Should read:
```
resource "azurerm_resource_group_policy_assignment" "example" {
  name                 = "example"
  resource_group_id    = azurerm_resource_group.example.id
  policy_definition_id = azurerm_policy_definition.example.id
}
```

Additionally, there are no examples on how to pass in parameters, which makes the document unclear / incomplete.

### PR DESCRIPTION
**Pull request by Sheetalsingh23**
Fixed #12542 Changed the incorrect code sample.

### CODE REVIEW

**Comment by Sheetalsingh23** [comment]
@gswallow can you please review the PR?

**Review by stephybun — CHANGES_REQUESTED** [review]
Thanks for this @Sheetalsingh23! Once the formatting issue caught by the website linting is fixed this can be merged.

File: website/docs/r/resource_group_policy_assignment.html.markdown [hunk 1]
```diff
@@ -39,9 +39,18 @@ POLICY_RULE
 }
 
 resource "azurerm_resource_group_policy_assignment" "example" {
-  name                 = "example"
-  resource_group_id    = azurerm_resource_group.example.id
+  name = "example"
+  resource_group_id = azurerm_resource_group.example.id
   policy_definition_id = azurerm_policy_definition.example.id
```
Comment (katbyte) [inline website/docs/r/resource_group_policy_assignment.html.markdown:44]:
```suggestion
  name                         = "example"
  resource_group_id  = azurerm_resource_group.example.id
  policy_definition_id = azurerm_policy_definition.example.id
```

**Review by stephybun — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**